participation/membership-tiers.md

```diff
@@ -1,2 +1,9 @@
 |Corda Network Foundation|[Document history]({{ site.github.repository_url }}/blame/master/{{page.path}})|
 
+Currently there are two membership tiers for Corda Network:
+(1) Full participant
+(2) Identity-Only / sponsored participant
+
+A **full participant** will usually sign contracts directly with R3. They will usually manage their own nodes and be part of  tightly-knit business network(s). They will receive full customer support from the Network Operator, can be a member of multiple business networks, will have voting rights and can stand for the Foundation's Board of directors.  
+
+In contrast, **identity-only participants** will typically be seen in much larger, disparate business networks where participants only infrequently transact on Corda Network. They only need an identity for one node application, do not receive any customer support, do not have voting rights and cannot stand for the Foundation's board of directors. They are issued with a certificate by the Network Doorman, they will be on the network map and a lower level of identity checking will have been done, given that they are part of a sponsored model. 
```

Comment:
what is a "node application" - is that just an "application"?
"are expected to receive on-boarding support from the business network or application provider, instead of the Network Operator",

is Board capitalised or not (c.w. previous paragraph)?

Reply:
yes just application. Have amended sentence to include your support point. Board not capitalised, have updated and will reflect throughout.